catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/source/WfsSource.java

```diff
@@ -903,38 +901,44 @@ private Pair<ExtendedGetFeatureType, Map<String, Serializable>> buildGetFeatureR
    * <p>Returns a properties map that includes information indicating if the sort parameters were
    * changed to the default.
    */
-  private Map<String, Serializable> setSortBy(SortBy sortBy, QName key, QueryType wfsQuery)
-      throws UnsupportedQueryException {
-    SortByType sortByType = buildSortBy(key, sortBy);
+  private void setSortBy(
+      SortBy sortBy, QName key, QueryType wfsQuery, Set<SourceProcessingDetails> details) {
+    SortByType sortByType;
+    String message;
+
+    if (this.disableSorting) {
+      message = "Source has sorting disabled.";
+      details.add(new ProcessingDetailsImpl(this.getId(), null, message));
+      LOGGER.debug(message);
+      return;
+    }
+
+    sortByType = buildSortBy(key, sortBy);
     if (isSortByValid(sortByType)) {
       logSortBy(sortByType);
       wfsQuery.setSortBy(sortByType);
-      return null;
+      return;
     }
 
-    Map<String, Serializable> properties = new HashMap<>();
-    properties.put(
-        Wfs11Constants.UNSUPPORTED_SORT_BY_REMOVED + "." + getId(),
-        sortBy.getPropertyName().getPropertyName());
+    message =
+        String.format(
+            "Source does not support specified sort property [%s]",
+            sortBy.getPropertyName().getPropertyName());
 
     if (defaultSortName == null || defaultSortOrder == null) {
-      LOGGER.debug(
-          "Skipping sorting because the query-supplied sort property [{}] is not supported and the default sort property and default sort order are not set.",
-          sortBy.getPropertyName().getPropertyName());
-      return properties;
+      details.add(new ProcessingDetailsImpl(this.getId(), null, message));
```

Comment:
Looks like this is called in every case, so we can move it immediately after the last message is constructed.